src/software/ai/evaluation/calc_best_shot.h

```diff
@@ -46,8 +25,7 @@ std::optional<Shot> calcBestShotOnGoal(const Point &goal_post_neg,
  * shot (this is the total angle between the obstacles on either side of the shot
  * vector). If no shot is possible, returns `std::nullopt`
  */
-std::optional<Shot> calcBestShotOnGoal(const Point &goal_post_neg,
-                                       const Point &goal_post_pos, const Point &p,
+std::optional<Shot> calcBestShotOnGoal(const Segment &goal_post, const Point &shot_origin,
                                        const std::vector<Robot> &obstacles);
```

Comment:
change to robot_obstacles to be consistent with comment and cpp

Reply:
Done.